src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -121,7 +121,8 @@ private function parseShortOptionSet($name)
         $len = \strlen($name);
         for ($i = 0; $i < $len; ++$i) {
             if (!$this->definition->hasShortcut($name[$i])) {
-                throw new RuntimeException(sprintf('The "-%s" option does not exist.', $name[$i]));
+                $encoding = mb_detect_encoding($name);
```

Comment:
we muse use strict mode, otherwise the detection is useless: `mb_detect_encoding($name, null, true);`